app.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function(){
```

Comment:
Please update this to ES6 syntax i.e. 
```suggestion
export function(){
```

Reply:
I already did it.